test/alternator/test_cql_rbac.py

```diff
@@ -631,6 +631,24 @@ def test_rbac_updatetable(dynamodb, cql):
                     tab.meta.client.update_table(TableName=tab.name,
                         BillingMode='PAY_PER_REQUEST')
 
+# A test for API operations that do not require any permissions, so can be
```

Comment:
I'm not sure if that's a good move, perhaps these operations should require at least read permissions. The motivation for introducing authz to Alternator was coming from a client who wishes to run multi-tenant clusters, but with this implementation all the cluster users would be able to see each others tables etc., right?

Reply:
We can discuss this, and it will be easy for me to change if we decide differently (this is also why I also documented this in the cover letter and in compatibility.md, hoping that @tzach and others will also complain if it bothers them).
My inspiration for this decision was the fact that in CQL, "DESCRIBE TABLE" or "DESCRIBE TABLES" does not require read permissions or any other permissions, and if I understand correctly, neither does reading from system tables. So I thought that Alternator operations which just lets you list tables or describe them can also be permission-less.

Reply:
@hopugop do you have any suggestions here?

Reply:
DESCRIBE and system table read access should not be restricted - otherwise we can break edge/unforeseen cases compatibility.  CQL' s way of handling it as described by @nyh sounds appropriate.

Reply:
Awesome, thanks for the feedback.